contribute.html

```diff
@@ -0,0 +1,434 @@
+					<div class="col-md-4"><a href="images/contribute/improve-docs.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/improve-docs.jpg" alt="VNC Server" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Expand DietPi Docs</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> This is an ever growing item - the more, the better</p>
+							<p>Looking for samples, documentation etc.</p>
+						</div>
+					</div>
+				</div>
+				<!-- End details for portfolio project 6 -->
+				<!-- Start details for portfolio project 7 -->
+				<div id="slidingdiv7" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/new-features.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/new-features.jpg" alt="NoMachine" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>New feature suggestions & Software titles</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> Let's add links to GitHub add issues (similar with what we have now on DietPi, we should also create on DietPi-Docs, and DietPi-Website)</p>
+							<p>Suggest ways to improve the application.</p>
+						</div>
+					</div>
+				</div>
+				<!-- End details for portfolio project 7 -->
+				<!-- Start details for portfolio project 8 -->
+				<div id="slidingdiv8" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/expand-code.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/expand-code.jpg" alt="XRDP" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Extend DietPi</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> This section is all about the code. Mainly if they have the expertise to help us fixing / enhancing DietPi project</p>
+							<p>Bring your contributions to DietPi projects.</p>
+							<p>DietPi project.</p>
+						</div>
+					</div>
+				</div>
+				<!-- End details for portfolio project 8 -->
+				<!-- Start details for portfolio project 9 -->
+				<div id="slidingdiv9" class="toggleDiv row single-project">
+					<div class="col-md-4"><a href="images/contribute/be-part-community.jpg" target="_blank" rel="noopener">
+						<img src="images/contribute/be-part-community.jpg" alt="RealVNC" loading="lazy"></a></div>
+					<div class="col-md-8">
+						<div class="project-description">
+							<div class="project-title clearfix">
+								<h3>Use DietPi Forum</h3>
+								<span class="show_hide close"></span>
+							</div>
+							<p><b>Core idea:</b> Browse, found things to contribute and help</p>
```

Comment:
`</b> Browse, found things to contribute and help</p>` -> `</b> Browse found things to contribute and help.</p>`